cv19ResSupportV3/V3/Gateways/ICaseNotesGateway.cs

```diff
@@ -6,7 +6,7 @@ namespace cv19ResSupportV3.V3.Gateways
     public interface ICaseNotesGateway
     {
         ResidentCaseNote PatchCaseNote(int helpRequestId, int residentId, string caseNote);
-        ResidentCaseNote CreateCaseNote(int helpRequestId, int residentId, string caseNote);
+        ResidentCaseNote CreateCaseNote(int residentId, int helpRequestId, string caseNote);
         ResidentCaseNote UpdateCaseNote(int helpRequestId, int residentId, string caseNote);
```

Comment:
Is this and patch is something we need to look at as well?

Reply:
look like its correct in the api